### ISSUE TITLE
Update for R 4.2.0 CRAN compatibility

### ISSUE THREAD

**Issue by jknowles** [jknowles/merTools#123]
CRAN Check identifies new issues:

NOTE:
```
checking R code for possible problems (11.8s)
   Found if() conditions comparing class() to string:
   File 'merTools/R/subBoot.R': if (class(merMod) == "lmerMod") ...
   File 'merTools/R/subBoot.R': if (class(merMod) == "glmerMod") ...
   File 'merTools/R/subBoot.R': if (class(merMod) == "lmerMod") ...
   File 'merTools/R/subBoot.R': if (class(merMod) == "glmerMod") ...
   Use inherits() (or maybe is()) instead.
```

**Comment by jknowles** [comment]
For documentation:

```

Warning: @inheritParams failed in topic "lmerModList", "blmerModList",
"glmerModList", and "bglmerModList".
✖ All parameters are already documented; none remain to be inherited.

```

**Comment by jknowles** [comment]
Closed  by 3abd5fc11dd42
